java/ql/src/experimental/Security/CWE/CWE-352/JsonpInjectionLib.qll

```diff
@@ -6,67 +6,69 @@ import semmle.code.java.dataflow.DataFlow
 import semmle.code.java.dataflow.FlowSources
 import semmle.code.java.frameworks.spring.SpringController
 
-/** Taint-tracking configuration tracing flow from user-controllable function name jsonp data to output jsonp data. */
+/** Taint-tracking configuration tracing flow from untrusted inputs to verification of remote user input. */
 class VerificationMethodFlowConfig extends TaintTracking::Configuration {
   VerificationMethodFlowConfig() { this = "VerificationMethodFlowConfig" }
 
   override predicate isSource(DataFlow::Node src) { src instanceof RemoteFlowSource }
 
   override predicate isSink(DataFlow::Node sink) {
-    exists(MethodAccess ma, BarrierGuard bg |
+    exists(MethodAccess ma |
```

Comment:
There might have been a misunderstanding between us regarding this `BarrierGuard`. I didn't mean to remove it, instead I was proposing that you should add more tests that make it required. Or did you want to change it so that all method calls that match the signature are verification methods?

For example, I think your current setup would make `logReferer` also a verification method, wouldn't it?
```
String referer = getReferer();
logReferer(referer);
...
```

Reply:
Thank you for your review. Regarding the issue here, I also think I have a misunderstanding. Can we have a place to solve it quickly? Such as email.